frontend/pages/index/contribute.vue

```diff
@@ -22,7 +22,13 @@
           >
             Please draw at least one feature from the map
           </b-badge>
-          <b-badge v-else show variant="primary" class="p-2">
+          <b-badge
```

Comment:
@bomiku I think this should be another branch called `fix/mobile-<something>` since it's unrelated to the other updates.

You might think that it's unnecessary because it's just 1 file change, and it's actually not an issue here. The problem is if you have more than 1 of this "small" supposed-to-be independent branches, but you bundled them into another since you only changed so little.

Make it a habit to break down unrelated updates as much as possible. You'll know if the branch name doesn't make sense, for example here.

Just some food for thought.